geo/src/lib.rs

```diff
@@ -264,10 +264,10 @@ extern crate log;
 /// Mean radius of Earth in meters
 /// This is the value recommended by the IUGG:
 /// Moritz, H. (2000). Geodetic Reference System 1980. Journal of Geodesy, 74(1), 128–133. doi:10.1007/s001900050278
-/// "Derived Geometric Constants: mean radius" (p133)
-/// https://link.springer.com/article/10.1007%2Fs001900050278
-/// https://sci-hub.se/https://doi.org/10.1007/s001900050278
-/// https://en.wikipedia.org/wiki/Earth_radius#Mean_radius
+/// "Derived Geometric Constants: **R1: mean radius**" (p132)
```

Comment:
```suggestion
/// "Derived Geometric Constants: **R1: mean radius**" (p131)
```

<img width="455" alt="Screenshot 2025-01-10 at 14 08 18" src="https://github.com/user-attachments/assets/6cd5189d-5dbb-42df-90d3-9fc02229157c" />

Maybe it's also on p132 (i haven't found it), but I think the table on p131 is nice and readable.